e2e/cypress/tests/codebase.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { loginBeforeEach } from "./setup.spec";
+import "cypress-file-upload";
+
+//login
+describe("Login", () => {
+  it("should log into comses homepage with test user", () => {
+    loginBeforeEach("test_user", "123456");
+  });
+});
+
+
+describe("Visit codebases page", () => {
+  //codebases PAGE
+
+  it("should visit the codebases page", () => {
+    cy.visit("/codebases");
+    assert(cy.get("h1").contains("Computational Model Library"));
+  });
+
+  it("should  be able to download a codebase", () => {
+    cy.visit("/codebases");
+    cy.get(".search-result").first().find("a").first().click();
```

Comment:
How brittle is this criteria? Is the first a href after a `.search-result` CSS class targeted enough to always get the first codebase? Or should we make all codebase results more explicit with a CSS class or identifier and target that explicitly?

Reply:
I think this and the above comment are the same issue, should probably just use data attributes to target this